lib/MaxMind/DB/Writer/Tree.pm

```diff
@@ -707,10 +730,29 @@ also handle unsigned 64-bit and 128-bit integers if they are passed as
 L<Math::UInt128|Math::Int128> objects.
 
 C<$additional_args> is a hash reference containing additional arguments that
-change the behavior of the insert. Currently, the only supported argument is
-C<force_overwrite>. This causes the object-wide C<merge_record_collisions>
-setting to be ignored for the insert, causing C<$data> to overwrite any
-existing data for the network.
+change the behavior of the insert. The following arguments are supported:
+
+=over 2
+
+=item * C<force_overwrite>
+
+This causes the tree's C<merge_record_collisions> setting to be ignored
+for the insert, causing C<$data> to overwrite any existing data for the
+network.
+
+=item * C<merge_subrecord_only_if_parent_exists>
+
+When merging the data record with an existing data record, do not create new
```

Comment:
Not sure I understand. What language are you proposing? In particular, it is not limited to a hashref within a hashref. It can also be a hashref within an arrayref.